guides/hosting/infrastructure/database-cluster.md

```diff
@@ -0,0 +1,41 @@
+# Database Cluster
+
+To scale Shopware beyond the horizon, we recommend you to use a database cluster. A database cluster consists of multiple read-only servers that are managed by a single primeray instance. Shopware comes in default with an read write splitting of SQL queries. When an Insert/Update/Delete query is executed, the query is distributed to the primary server and the connection uses only the primary until the end of the connection, this will be also triggered by the ``executeStatement`` in the [DebugStack decoration](https://github.com/shopware/platform/blob/v6.4.11.1/src/Core/Profiling/Doctrine/DebugStack.php#L48).
```

Comment:
Is "beyond the horizon" the right choice of words? I mean Udo Lindenberg is not wrong with that and you want to express to overcome a limitation, but I would still think about it. Maybe a "even further" is a better one. Maybe you want to express instead that there are a lot of performance tweaks you can do and one of them is to upgrade your single instance to a clustered database.

Reply:
When i am in the flow i just write anything at once forgot to look later again into 😂